src/modules/common/constants.js

```diff
@@ -0,0 +1,4 @@
+import { detect } from 'detect-browser'
+const browser = detect()
+
+export const positionStickyDisabled = browser && browser.name === 'edge' && /^16\..*/.test(browser.version)
```

Comment:
Imporing this file can have unintendet side effects because detect() is called.

I would propose to have a Platform class which calls detect() in the constructor and has a function isPositionStickyDisabled

Reply:
@maxammann Hmm... should we have the platform class instance then somewhere in the state?
Otherwise we would create a platform instance in each stylesheet, and therefore call detect() many times, which could impact performance...

Reply:
Then it would make sense to add a PlatformProvider as react context. State doesn't really make sense beause its static data (also class instances shouldn't be in the state. Only obejcts).

Reply:
Done